Makefile

```diff
@@ -93,14 +89,16 @@ deb-build-geoserver-geofence: war-build-geoserver-geofence
 	cd geoserver; \
 	../mvn clean package deb:package -PdebianPackage --pl webapp
 
-deb-build-deps:
+deb-build-georchestra: war-build-georchestra build-deps deb-build-geoserver
+	./mvn package deb:package -pl atlas,catalogapp,cas-server-webapp,downloadform,security-proxy,header,mapfishapp,extractorapp,analytics,geoserver/webapp,ldapadmin,geonetwork/web,geowebcache-webapp -PdebianPackage -DskipTests
+
+# Base geOrchestra config and common modules
+build-deps:
 	./mvn -Dmaven.test.failure.ignore clean install --non-recursive
 	./mvn clean install -pl config -Dmaven.javadoc.failOnError=false
 	./mvn clean install -pl commons,epsg-extension,ogc-server-statistics -Dmaven.javadoc.failOnError=false
-
-deb-build-georchestra: war-build-georchestra deb-build-deps deb-build-geoserver
-	./mvn package deb:package -pl atlas,catalogapp,cas-server-webapp,downloadform,security-proxy,header,mapfishapp,extractorapp,analytics,geoserver/webapp,ldapadmin,geonetwork/web,geowebcache-webapp -PdebianPackage -DskipTests
-
+	cd config/; \
```

Comment:
You are not forced to change directory, you could either use one of the following maven options: 

```
 -pl,--projects <arg>                   Comma-delimited list of specified
                                        reactor projects to build instead
                                        of all projects. A project can be
                                        specified by [groupId]:artifactId
                                        or by its relative path.

 -f,--file <arg>                        Force the use of an alternate POM
                                        file (or directory with pom.xml).
```